lib/kino/livebook_output.ex

```diff
@@ -0,0 +1,58 @@
+defmodule Kino.LivebookOutput do
+  @moduledoc """
+  A number of output formats supported by Livebook.
+  """
+
+  @typedoc """
+  Livebook cell output may be one of these values
+  and is rendered accordingly.
+  """
+  @type t ::
```

Comment:
@josevalim we discussed using a `%Kino.Output{}` struct, but I realised the output type must be recognisable in Livebook itself, so I think it's ok to stick to tuples (and this one-of type is also neat for the plain string output).